src/Okta.Idx.Sdk/IdxClient.cs

```diff
@@ -1902,5 +1902,30 @@ public async Task<PollResponse> PollAuthenticatorPushStatusAsync(IIdxContext idx
                 },
                 challengeResponse);
         }
+
+        /// <inheritdoc/>
+        public async Task<PasswordRequiredResponse> CheckIsPasswordRequiredAsync(string state = null, CancellationToken cancellationToken = default)
+        {
+            var idxContext = await this.InteractAsync(state, cancellationToken);
+            return await CheckIsPasswordRequiredAsync(idxContext, cancellationToken);
+        }
+
+        /// <inheritdoc/>
+        public async Task<PasswordRequiredResponse> CheckIsPasswordRequiredAsync(IIdxContext idxContext, CancellationToken cancellationToken = default)
+        {
+            if (idxContext == null)
+            {
+                throw new ArgumentNullException(nameof(idxContext));
+            }
+
+            var introspectResponse = await this.IntrospectAsync(idxContext, cancellationToken);
+
+            return new PasswordRequiredResponse
+            {
+                Context = idxContext,
+                IsPasswordRequired = IsRemediationRequireCredentials("identify", introspectResponse),
+                Response = introspectResponse,

```

Comment:
is there any reason why we're returning the raw response? We usually don't include it as part of other responses (by design).

Reply:
I thought it would be useful for debugging if it became necessary.  I don't feel strongly about keeping it.